src/components/action-bar/action-bar.scss

```diff
@@ -0,0 +1,66 @@
+/**
+* @prop --action-bar-item-text-color: Text color of action bar items, defaults to `--contrast-1100`.
+* @prop --action-bar-item-icon-color: Color of the icons displayed on each action bar item. Defaults to the text color. To specify a color for an individual item, use the `iconColor` prop instead.
+* @prop --action-bar-item-max-width: Maximum width of a button in the action bar. Defaults to `10rem`. Keep in mind that the buttons should not appear too big.
+* @prop --action-bar-border-radius: Defines the roundness of the corners of the action bar. Defaults to `0`.
+* @prop --action-bar-background-color: Background color of the whole component. Defaults to `--contrast-100`.
+*/
+
+:host(limel-action-bar) {
+    --action-bar-item-height: 2rem;
+
+    --limel-action-bar-item-text-color: var(
+        --action-bar-item-text-color,
+        rgb(var(--contrast-1100))
+    );
+
+    box-sizing: border-box;
+
+    display: inline-flex;
+    align-items: center;
+    padding: 0.125rem 0.25rem;
+    max-width: 100%;
+    border-radius: var(--action-bar-border-radius);
+
+    background-color: var(
+        --action-bar-background-color,
+        rgb(var(--contrast-100))
+    );
+
+    gap: 0.25rem;
+    @media (pointer: coarse) {
+        gap: 0.5rem;
+    }
+}
+
+limel-action-bar-button {
+    min-width: 0;
+    display: absolute;
+}
+
+.visible-items {
+    display: inline-flex;
+    overflow-x: hidden;
```

Comment:
<img width="535" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/e4bb5b5a-4f98-4981-b644-23a74efa52ce">
This will cause a rendering problem for the shadows (hovered buttons).
We should conditionally set overflow while resizing the component. When the resize and calculations are done, remove the overflow.

Reply:
fixed now